templates/blockonomics_nojs_checkout.php

```diff
@@ -1,70 +1,81 @@
 <?php
+$blockonomics = new Blockonomics;
 $orders = get_option('blockonomics_orders');
-$order_id = $_REQUEST['show_order'];
-$crypto = $_REQUEST['crypto'];
-foreach ($orders[$order_id] as $addr => $order){
-  if ($order['crypto'] == $crypto) break;
-}
-?>
-<div ng-app="shopping-cart-demo">
-  <div ng-controller="CheckoutController">
+$crypto = isset($_REQUEST["crypto"]) ? $_REQUEST["crypto"] : "";
+$order_id = isset($_REQUEST["show_order"]) ? $_REQUEST["show_order"] : "";
+$qrcode = isset($_REQUEST['qrcode']) ? $_REQUEST['qrcode'] : "";
+
+if ($qrcode) {
+  $blockonomics->generate_qrcode($qrcode);
```

Comment:
This code should be there in *WC_Gateway_Blockonomics.php/handle_requests()* 
That is where we are handling all the routing logic

Reply:
Moved in f30f1d9